introduction.md

```diff
@@ -1,2 +1,5 @@
 I am a Time Lord.
-Code school path: Electives 
\ No newline at end of file
+Code school path: Electives
+*Git
+*Javascript
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?